index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/a70cf7601d.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="./css/styles.css" />
+    <title>Personal Portfolio</title>
+  </head>
+
+  <body>
+    <section>
+      <header class="header">
+        <nav class="nav">
+          <a href="">Welcome</a>
+          <i class="fas fa-bars"></i>
+        </nav>
+      </header>
+      <div class="hero-heading">
+        <h1>Hey there, I am Dennis.</h1>
```

Comment:
- [ ] Your headline design is matching the Figma almost perfectly. Well done. I'd like to point out though; that the color of the text in the `h1` is not completely black in Figma design. the actual color is `#172B4D`. Please change this as we want the two designs to match almost perfectly. This will build your CSS skills and ability to match a set design 🙏.
## Figma Design
![image](https://user-images.githubusercontent.com/18730106/143382330-4f5e9904-7408-4baa-ba8d-2610a9593aa8.png)
## Your Design
![image](https://user-images.githubusercontent.com/18730106/143382358-499557ee-4043-48ba-bba6-b6c50b0fa422.png)

Reply:
Fixed as per requirement/review. Thank you.